frontend/packages/kubevirt-plugin/src/types/ui/ui.ts

```diff
@@ -0,0 +1,8 @@
+export enum UIValidationType {
+  LENGTH,
+}
+
+export type UIValidation = {
+  type: UIValidationType;
+  settings: { [k: string]: any };
```

Comment:
should be optional for a generic type

Reply:
I would prefer to put all the known types here compared to not knowing what is inside. min/max

Reply:
ping

Reply:
Done!